src/Parser.h

```diff
@@ -6,14 +6,14 @@ found in the LICENSE file. See the AUTHORS file for names of contributors.
  */
 #ifndef SHORTLINK_SRC_PARSER_H_
 #define SHORTLINK_SRC_PARSER_H_
+#include <memory>
 #include <sstream>
 #include <string>
 
 namespace shortlink {
 
 // forward declare
 class RequestData;
-class SURL;
 
 // This class represents a operation that can translate raw message to
 // executable information of a request. Example:
```

Comment:
Maybe add in the comment that the raw data is not an HTTP request, but something that is part of the HTTP request.

Reply:
Added comments about it.